ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -9,21 +9,22 @@ import UIKit
 
 class TooltipView: UIView {
 
-    init(anchorView: UIView,
-         message: String,
+    init(message: String,
          title: String? = nil,
          textAlignment: NSTextAlignment,
          preferredArrowDirection: Tooltip.ArrowDirection,
          offset: CGPoint,
-         arrowMargin: CGFloat,
-         tokenSet: TooltipTokenSet) {
+         arrowMargin: CGFloat) {
+        guard let anchorView = Tooltip.shared.anchorView else {
```

Comment:
It was to address Harrie's comment above: "_this is also concerning that Tooltip.swift holds on the the anchorView and there is additional private let of anchorview inside the TooltipView?_"

If I understand correctly, since Tooltip itself is a singleton, the change is essentially doing the same thing and simplifies the logic?